src/components/inline_edit/inline_edit_title.tsx

```diff
@@ -28,6 +28,7 @@ export type EuiInlineEditTitleProps = EuiInlineEditCommonProps & {
   size?: EuiTitleSize;
   /**
    * Level of heading to be used for the title
```

Comment:
Our markdown parser doesn't add a `<br />` for newlines (only adds pargraphs for two newlines) so we need to use correct punctuation here so the two sentences don't run on together.
```suggestion
   * Level of heading to be used for the title.
```